pkg/fetcher/definition_test.go

```diff
@@ -150,3 +281,20 @@ func (m *mockErrReader) Get(ctx context.Context, key client.ObjectKey, obj runti
 func (m *mockErrReader) List(ctx context.Context, opts *client.ListOptions, list runtime.Object) error {
 	return m.err
 }
+
+type mockListReader struct {
```

Comment:
can we get rid of that: https://github.com/kubernetes-sigs/controller-runtime/pull/311 ?

Reply:
It's available since `controller-runtime` `v0.1.9`, while kubebuilder uses `v0.1.9`.